lib/foreman_theme_satellite/documentation.rb

```diff
@@ -51,7 +51,7 @@ module Documentation
       'LDAPAuthentication' => "#{ForemanThemeSatellite.documentation_root}/installing_satellite_server_in_a_connected_network_environment/configuring_external_authentication_satellite#Using_LDAP_satellite",
       'Bookmarks' => "#{ForemanThemeSatellite.documentation_root}/administering_red_hat_satellite/searching_and_bookmarking_admin",
       'Policydeploymentoptions' => "#{ForemanThemeSatellite.documentation_root}/managing_security_compliance#Managing_Compliance_Policies_security-compliance",
-      'CreatingSCAPcontent' => "#{ForemanThemeSatellite.documentation_root}/administering_red_hat_satellite/managing_security_compliance_admin#running-a-security-compliance-scan-on-demand_admin",
+      'CreatingSCAPcontent' => "#{ForemanThemeSatellite.documentation_root}/managing_security_compliance/running-a-security-compliance-scan-on-demand_security-compliance",
```

Comment:
I smell this will need backporting

Reply:
No backporting should be needed. The new guide is there only since Sat 6.14 (Foreman 3.6).
The original link should work just fine for 6.13.

Reply:
Technically this repository is for 6.15 and highe. 6.14 only lives downstream so a backport will be needed.